src/lib/stores/user.ts

```diff
@@ -134,7 +132,7 @@ export async function loadUser(): Promise<void> {
 
 export async function loadUserProfile() {
     // Get user public key from extension
-    const pubkey = await (window as any).nostr.getPublicKey();
+    const pubkey = ndk.signer?.pubkey || get(user)?.pubkey;
```

Comment:
@callebtc this introduced a bug:

<img width="737" alt="Screenshot 2025-05-01 at 17 50 41" src="https://github.com/user-attachments/assets/b91c8db8-912e-4826-8b3e-16c58cdd63dd" />